tools/releaser/scripts/breaking-changes.sh

```diff
@@ -1,37 +1,45 @@
 #!/bin/bash
 set -eu
-GOPATH=$(go env GOPATH)
 
 # Inputs:
-# API_DIFF_OLD_COMMIT: commit before the API changes to compare with. If not provided, script will fail with "unbound variable" error
-# API_DIFF_NEW_COMMIT: commit with the new API changes. If not provided, script will fail with "unbound variable" error
-# TARGET_BREAKING_CHANGES_FILE - file to save breaking changes
```

Comment:
Does script do not have any inputs right now?

Reply:
no.  The script now essentially uses the last release version (from extract-version.sh)